cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -134,7 +133,7 @@ public AppFabricHttpHandler(Authenticator authenticator, CConfiguration configur
     this.queueAdmin = queueAdmin;
     this.txClient = txClient;
     this.dsFramework =
-      new NamespacedDatasetFramework(dsFramework, new DefaultDatasetNamespace(configuration, Namespace.USER));
+      new NamespacedDatasetFramework(dsFramework, new DefaultDatasetNamespace(configuration));
```

Comment:
Are there places a DatasetFramework is used directly from the guice binding without this wrapper?  If not, since we're no longer passing in Namespace.USER or Namespace.SYSTEM, it would probably make sense to refactor so that the right implementation is bound and can be used directly without wrapping it.